src/Domain/FunctionsOfTime/PiecewisePolynomial.cpp

```diff
@@ -154,6 +154,30 @@ bool operator!=(const PiecewisePolynomial<MaxDeriv>& lhs,
   return not(lhs == rhs);
 }
 
+template <size_t MaxDeriv>
+std::ostream& operator<<(
+    std::ostream& os,
+    const PiecewisePolynomial<MaxDeriv>& piecewise_polynomial) noexcept {
+  const auto size = piecewise_polynomial.deriv_info_at_update_times_.size();
+
+  auto print_deriv_info =
+      [](std::ostream& local_os,
+         const FunctionOfTimeHelpers::StoredInfo<MaxDeriv + 1>& deriv_info) {
+        local_os << deriv_info.time << ":";
+        for (size_t i = 0; i < MaxDeriv; ++i) {
+          local_os << gsl::at(deriv_info.stored_quantities, i) << " ";
+        }
```

Comment:
Instead of just printing the time with a colon, maybe do `local_os << "t=" << deriv_info.time << ":"`. That way it's clear what is left of the colon.